vlsi/bwrc-env.yml

```diff
@@ -0,0 +1,7 @@
+mentor.mentor_home: "/tools/mentor"
```

Comment:
Can we put links to the license servers in the public repo?
Can these be moved to the cad-plugins repos?

Reply:
Well, these should not work for anyone who doesn't have BWRC computing resource access. I don't think paths to license servers is violating any NDA (@jwright6323 please weigh in)? 

The cad plugins should default to blank for these fields so that the user can override.

Reply:
I think it would be ideal to not have this be in Chipyard. I think from the outside users will think "what is BWRC" and ask questions about what this file is. 

To me this should be something that a downstream repo should have. Brainstorming... maybe have a submodule that holds BWRC configurations... or have this be an env variable that needs to be set where the env variable points to this file that is located somewhere and tell the Chipyard user that they need to point to an environment config file?

Reply:
I think having this in the cad-plugins directories makes sense, since each cad-plugin corresponds to a license server,

Reply:
bwrc-env should just live somewhere internal to Berkeley/BWRC IMO.

Reply:
Yeah, it's no longer part of this PR.